dask_deltatable/core.py

```diff
@@ -3,12 +3,14 @@
 import os
 from collections.abc import Sequence
 from functools import partial
-from typing import Any, cast
+from typing import Any, Callable, cast
 
+import dask
 import dask.dataframe as dd
 import pyarrow as pa
 import pyarrow.parquet as pq
 from dask.base import tokenize
+from dask.dataframe.io.parquet.arrow import ArrowDatasetEngine
```

Comment:
This import is a bit ugly but this class exposes our type mappers and this way we can align behavior